shared/auth/auth.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.28.1
-// 	protoc        v3.21.5
+// 	protoc-gen-go v1.25.0-devel
```

Comment:
I guess it works, but your stack looks a bit outdated :-)

Reply:
Sorry, I just used the default generator installed on my system to generate the files. Feel free to regenerate the files with more recent versions.